package.bzl

```diff
@@ -32,6 +32,7 @@ NPM_PACKAGE_SUBSTITUTIONS = select({
 BZL_DEFAULTS_ALLOW_PACKAGES = [
     ".github/local-actions",
     "",
+    "api-gen",
```

Comment:
FYI: If we end up shipping the `.bzl` files and `BUILD` files of `/api-gen` (which is not done here anyway), then we actually _cannot_ use the defaults and this line needs to be reverted/removed.

Reply:
Ah, good catch. Done.